content/minfraud/api-documentation/examples.mdx

```diff
@@ -0,0 +1,217 @@
+export const request = `
```

Comment:
I don't think this file is used anywhere

Reply:
Ah, this was an artifact from an idea I was exploring. I removed this file.